Sources/InfomaniakCore/Background/ClosureExpiringActivity.swift

```diff
@@ -19,13 +19,35 @@
 import CocoaLumberjackSwift
 import Foundation
 
-public enum BackgroundExecutor {
-    public typealias TaskCompletion = () -> Void
-    public static func executeWithBackgroundTask(_ block: @escaping (@escaping TaskCompletion) -> Void,
-                                                 onExpired: @escaping () -> Void) {
+public typealias TaskCompletion = () -> Void
+
+/// Something that can perform arbitrary short background tasks, with closure API.
+protocol ClosureExpiringActivable {
```

Comment:
Renamed `BackgroundExecutor` into `ClosureExpiringActivity` with a linked protocol `ClosureExpiringActivable` to move to something testable.